daemon/api_download.go

```diff
@@ -36,18 +40,33 @@ var snapDownloadCmd = &Command{
 	POST:     postSnapDownload,
 }
 
+var validRangeRegexp = regexp.MustCompile(`^\s*bytes=(\d+)-\s*`)
+
 // SnapDownloadAction is used to request a snap download
 type snapDownloadAction struct {
-	SnapName string `json:"snap-name,omitempty"`
+	SnapName string `json:"snap-name"`
 	snapRevisionOptions
+	ResumeStamp    string `json:"resume-stamp"`
+	resumePosition int64
+	NoBody         bool `json:"no-body"`
```

Comment:
I proposed it but "no-body" sounds too much like nobody, a minor thing, but I need to think a different name for this flag

Reply:
omitBody maybe?

Reply:
`only-headers` or `only-head` or perhaps just `head`?